foundations/introduction/motivation_and_mindset.md

```diff
@@ -168,3 +168,5 @@ This section contains helpful links to related content. It isn’t required, so
 *   [Improve your typing skills with Keybr.com](https://www.keybr.com/) Use this keyboard trainer if you find your typing speed is holding you back. We recommend spending 5 mins every day after you start your PC.
 *   [Practice typing with Monkeytype!](https://monkeytype.com/) A minimalistic, customizable typing website. Test yourself in various modes, track your progress and improve your typing speed. It has a great community and frequently receives new features. Even though it has so many features, the experience is still very polished. 
 *   [Why Procrastinators Procrastinate](https://waitbutwhy.com/2013/10/why-procrastinators-procrastinate.html) Learn about the Instant Gratification Monkey, Rational Decision Maker, Panic Monster, and how to navigate the Dark Playground.
+*   [Understanding what ChatGPT is doing and why it works](https://writings.stephenwolfram.com/2023/02/what-is-chatgpt-doing-and-why-does-it-work/) Learn about the design of AI LLM models and what it is they're actually doing. This is a long, somewhat technical read, but well worth understanding the "magic" behind ChatGPT or Copilot.
+*   [ChatGPT is good at its designed task](https://www.pbs.org/newshour/science/analysis-chatgpt-is-great-at-what-its-designed-to-do-youre-just-using-it-wrong) Another article about what is happening behind the scenes, but more approachable and less technical.
```

Comment:
After some thought, I'm good with mentioning the idea of AI prompting as you have it in the current bullet point.

But I'm not sure we need/want to send learners to read about ChatGPT further. I think I'd feel best about removing these two links.